server/ansible/roles/pbench-server-crontab-enable/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: "enable crontab"
+  command:
+    cmd: crontab {{ pbench_server_install_dir }}/lib/crontab/crontab
```

Comment:
Shouldn't we be specifying a user name (either with a -u switch on the command or in a user: entry in the YAML)?

Reply:
Yes.